WooCommerce/Classes/ViewModels/OrderDetailsViewModel/OrderDetailsDataSource+Cells.swift

```diff
@@ -0,0 +1,257 @@
+
+        cell.accessibilityTraits = .staticText
+        cell.accessibilityLabel = NSLocalizedString(
+            "This order is using extensions to calculate shipping. The shipping methods shown might be incomplete.",
+            comment: "Accessibility label for the Shipping notice")
+        cell.accessibilityHint = NSLocalizedString("Shipping notice about the order",
+                                                    comment: "VoiceOver accessibility label for the shipping notice about the order")
+    }
+
+    func configureNewNote(cell: LeftImageTableViewCell) {
+        cell.leftImage = Icons.addNoteIcon
+        cell.labelText = Titles.addNoteText
+
+        cell.accessibilityTraits = .button
+        cell.accessibilityLabel = NSLocalizedString(
+            "Add a note",
+            comment: "Accessibility label for the 'Add a note' button"
+        )
+
+        cell.accessibilityHint = NSLocalizedString(
+            "Composes a new order note.",
+            comment: "VoiceOver accessibility hint, informing the user that the button can be used to create a new order note."
+        )
+    }
+
+    func configureOrderNoteHeader(cell: OrderNoteHeaderTableViewCell, at indexPath: IndexPath) {
+        guard let noteHeader = noteHeader(at: indexPath) else {
+            return
+        }
+
+        cell.dateCreated = noteHeader.toString(dateStyle: .medium, timeStyle: .none)
+    }
+
+    func configureOrderNote(cell: OrderNoteTableViewCell, at indexPath: IndexPath) {
+        guard let note = note(at: indexPath) else {
+            return
+        }
+
+        cell.isSystemAuthor = note.isSystemAuthor
+        cell.isCustomerNote = note.isCustomerNote
+        cell.author = note.author
+        cell.dateCreated = note.dateCreated.toString(dateStyle: .none, timeStyle: .short)
+        cell.contents = orderNoteAsyncDictionary.value(forKey: note.noteID)
+    }
+
+    func configurePayment(cell: PaymentTableViewCell) {
+        let paymentViewModel = OrderPaymentDetailsViewModel(order: order)
+        cell.configure(with: paymentViewModel)
+    }
+
+    func configureCustomerPaid(cell: TwoColumnHeadlineFootnoteTableViewCell) {
+        let paymentViewModel = OrderPaymentDetailsViewModel(order: order)
+        cell.leftText = Titles.paidByCustomer
+        cell.rightText = paymentViewModel.paymentTotal
+        cell.footnoteText = paymentViewModel.paymentSummary
+    }
+
+    func configureDetails(cell: WooBasicTableViewCell) {
+        cell.bodyLabel?.text = Titles.productDetails
+        cell.bodyLabel?.applyBodyStyle()
+        cell.accessoryImage = nil
+        cell.accessoryType = .disclosureIndicator
+        cell.selectionStyle = .default
+    }
+
+    func configureRefund(cell: TwoColumnHeadlineAttributedFootnoteTableViewCell, at indexPath: IndexPath) {
+//        let row = rowAtIndexPath(indexPath)
+        let condensedRefund = condensedRefunds[indexPath.row - 2] // TODO-thuy: minus two should be constants
```

Comment:
This needs refactored into `lookUpCondensedRefunds(_ indexPath)`. I had so much code in here already I didn't want to add more. I can convert this now if you prefer. 

An order can have multiple condensed refunds. In order to configure the correct row, I have to look up the correct refund. It's currently subtracting out 2 rows because the refunds are listed below the Payment Summary cell and the Paid by Customer cell are added to this section first.

Reply:
Got it, the TODO sounds good 👍